api/topics/monitor/new.go

```diff
@@ -0,0 +1,27 @@
+package monitor
+
+import (
+	"fmt"
+
+	"github.com/go-redis/redis"
+
+	"github.com/arkhn/fhir-river/api/topics"
+)
+
+type BatchController struct {
+	rdb *redis.Client
+	topics.Controller
```

Comment:
when using struct composition like this (which I find elegant btw) I think the good practice is to put the embedded structs on top like this:
```go
type BatchController struct {
	topics.Controller

	rdb *redis.Client
}
```
not a big deal anyway